pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/AbstractCompiler.java

```diff
@@ -16,47 +16,10 @@
 package com.linkedin.pinot.pql.parsers;
 
 import com.linkedin.pinot.common.request.BrokerRequest;
-import org.antlr.runtime.RecognitionException;
-import org.antlr.runtime.tree.CommonTree;
-import org.json.JSONObject;
-
-
-public abstract class AbstractCompiler {
-
-  public AbstractCompiler() {
-    super();
-  }
-
-  public abstract JSONObject compile(String expression) throws RecognitionException;
-
-  public abstract BrokerRequest compileToBrokerRequest(String expression);
-
-  public abstract String getErrorMessage(RecognitionException error);
-
-  protected void printTree(CommonTree ast) {
-    print(ast, 0);
-  }
-
-  private void print(CommonTree tree, int level) {
```

Comment:
Is print/printTree() implemented elsewhere? If not, would be nice to have that feature.